Makefile.generated_files

```diff
@@ -12,8 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
```

Comment:
Yes, the [default shell is `/bin/sh`](https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html#Choosing-the-Shell). I wonder if we should enforce this `bash`-ness, or figure out what is bash specific and change it.